cypress/components/menu/menu.cy.tsx

```diff
@@ -1,13 +1,16 @@
 import React from "react";
-import Menu from "../../../src/components/menu/menu.component";
-import MenuItem from "../../../src/components/menu/menu-item/menu-item.component";
-import MenuDivider from "../../../src/components/menu/menu-divider/menu-divider.component";
-import MenuSegmentTitle from "../../../src/components/menu/menu-segment-title/menu-segment-title.component";
-import MenuFullscreen from "../../../src/components/menu/menu-full-screen/menu-full-screen.component";
-import ScrollableBlock from "../../../src/components/menu/scrollable-block/scrollable-block.component";
+import {
+  Menu,
+  MenuProps,
+  MenuItem,
+  MenuWithChildren,
+  MenuDivider,
+  MenuDividerProps,
```

Comment:
Feel free to do this if you have the time, i'd group these imports together, so every component is followed by their props

Reply:
Like this `Menu
MenuProps
MenuItem
MenuWithChildren
MenuDivider
MenuDividerProps
MenuSegmentTitle
MenuFullscreenProps
ScrollableBlockProps` apologies I didn't suggest it grouped like this but I was typing it out whilst just looking at the list